docs/contributing/RELEASING.md

```diff
@@ -30,7 +33,8 @@ RELEASING TAX-CALCULATOR CONDA PACKAGES
 
 --> create release X.Y.Z on GitHub using master branch
 
---> run `pbrelease Tax-Calculator taxcalc X.Y.Z` [to build and upload packages]
+--> Create new package on Conda-Forge for release X.Y.Z
+    --> open a PR to github.com/conda-forge/taxcalc-feedstock where you change the `recipe/meta.yaml` file by updating (1) the version number to X.Y.Z and (2) the checksum to reflectr the checksum for the tarball for release X.Y.Z in the Tax-Calculator GitHub repo
```

Comment:
minor typo reflectr -> reflect

Reply:
Thanks for the catch -- just fixed this.